policy/modules/kernel/devices.te

```diff
@@ -385,6 +385,6 @@ files_associate_tmp(device_node)
 #
 
 allow devices_unconfined_type self:capability sys_rawio;
-allow devices_unconfined_type device_node:blk_file { manage_blk_file_perms relabelfrom relabelto map execute quotaon mounton audit_access execmod watch };
-allow devices_unconfined_type device_node:chr_file { manage_chr_file_perms relabelfrom relabelto map execute quotaon mounton execmod audit_access watch };
```

Comment:
Seems you dropped the permission `map`, its quite uncommon, but e.g. `dev_read_sound()` grants it

Reply:
Agreed.

Reply:
Very good catch, fixed.